package.json

```diff
@@ -25,8 +25,11 @@
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
     "release": "release-it",
     "example": "yarn --cwd example",
+    "example2": "yarn --cwd example2",
```

Comment:
please delete this, plus `pods2` and `bootstrap2`

Reply:
done